README.md

```diff
@@ -35,25 +35,50 @@ For more information:
 * [Wiki: Compare Fluture to similar libraries][wiki:similar]
 * [Video: Monad a Day - Futures by @DrBoolean][5]
 
-## Usage
+## Installation
+
+### With NPM
+
+```console
+$ npm install --save fluture
+```
+
+### Bundled from a CDN
+
+To load Fluture directly into a browser, a code pen, or [Deno][], use one of
+the following downloads from the JSDelivr content delivery network. These are
+single files that come with all of Fluture's dependencies pre-bundled.
+
+- [Fluture Script][]: A JavaScript file that adds `Fluture` to the global
+  scope. Ideal for older browsers and code pens.
+- [Fluture Script Minified][]: The same as above, but minified.
+- [Fluture Module][]: An EcmaScript module with named exports. Ideal for Deno
+  or modern browsers.
+- [Fluture Module Minified][]: A minified EcmaScript module without TypeScript
+  typings. Not recommended for Deno.
+
+[Fluture Script]: https://cdn.jsdelivr.net/gh/fluture-js/Fluture@12.3.0/dist/bundle.js
+[Fluture Script Minified]: https://cdn.jsdelivr.net/gh/fluture-js/Fluture@12.3.0/dist/bundle.min.js
+[Fluture Module]: https://cdn.jsdelivr.net/gh/fluture-js/Fluture@12.3.0/dist/module.js
+[Fluture Module Minified]: https://cdn.jsdelivr.net/gh/fluture-js/Fluture@12.3.0/dist/module.min.js
 
-> `npm install --save fluture`
+## Usage
 
 ### EcmaScript Module
 
 Fluture is written as modular JavaScript.
 
-- On Node 13 and up, you can load Fluture directly as shown in the example.
-- On Node 12, you'll need to run with `node --experimental-modules`.
-- On Node versions below 12, you can use the [esm loader][esm]. Alternatively,
-  you can use the [CommonJS Module](#commonjs-module).
+- On Node 13 and up, Fluture can be loaded directly as shown in the example.
+- On Node 12, the `--experimental-modules` flag must be provided.
+- On Node versions below 12, the [esm loader][esm] can be used. Alternatively,
+  there is a [CommonJS Module](#commonjs-module) available.
 - Modern browsers can run Fluture directly. If you'd like to try this out,
   I recommend installing Fluture with [Pika][] or [Snowpack][]. You can also
-  try the [bundled module](#global-bundle-cdn) to avoid a package manager.
-- For older browsers, you can use a bundler such as [Rollup][] or WebPack.
-  Fluture doesn't use ES5+ language features, so the source does not have to
-  be transpiled. Alternatively, you can use the
-  [CommonJS Module](#commonjs-module).
+  try the [bundled module](#bundled-from-a-cdn) to avoid a package manager.
+- For older browsers, use a bundler such as [Rollup][] or WebPack. Fluture
+  doesn't use ES5+ language features, so the source does not have to be
+  transpiled. Alternatively, there is a [CommonJS Module](#commonjs-module)
+  available.
```

Comment:
What do you mean by ES5+ language features, exactly? Fluture may not use ES5 language features, but I would be surprised to discover that it doesn't use any of the functions and methods specified in ES5 ([`Object.keys`][1], for example).


[1]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys

Reply:
> Fluture may not use ES5 language features

I'm talking about *post* ES5 language features, as (perhaps badly) indicated by the plus symbol.

> What do you mean by ES5+ language features, exactly?

I mean language features specifically, as opposed to newly added API's. So syntax.
As for newly added APIs, there's a comment about them further down in the README:

https://github.com/fluture-js/Fluture/pull/431/files#diff-04c6e90faac2675aa89e2176d2eec7d8R102-R103

Reply:
To me, _ES5+_ means `>= ES5` rather than `> ES5`. Perhaps you could describe Fluture as _ES5-compatible_ to avoid confusion.

Reply:
How is this? https://github.com/fluture-js/Fluture/pull/431/commits/2e719949311f88667921dac5a4550f8947a507b6

Reply:
That commit looks good. :)